samples-lib/src/main/java/com/esri/arcgismaps/sample/sampleslib/DownloaderActivity.kt

```diff
@@ -224,43 +233,38 @@ abstract class DownloaderActivity : AppCompatActivity() {
                 emit(LoadStatus.FailedToLoad(it))
                 return@flow
             }
+            // set up the file at the download path
+            val destinationFilePath = provisionLocation.path + File.separator + portalItem.name
```

Comment:
There are better options to create this path:
1. Use the `File` API - `val destinationFilePath = File(provisionLocation.path, portalItem.name).toString()`
2. ``val destinationFilePath = "${provisionLocation.path}File.separator${portalItem.name}"`